snuba/cli/optimize.py

```diff
@@ -35,16 +35,17 @@
 @click.option("--log-level", help="Logging level to use.")
 @click.option(
     "--parallel",
+    "default_parallel_threads",
     type=click.IntRange(1, 2),
```

Comment:
For now I kept this because I don't want the cron job to start with too many threads